spikeinterface/sorters/ironclust/ironclust.py

```diff
@@ -124,11 +150,16 @@ class IronClustSorter(BaseSorter):
 
     @classmethod
     def is_installed(cls):
+        if cls.is_compiled:
+            return True
         return check_if_installed(cls.ironclust_path)
 
-    @staticmethod
-    def get_sorter_version():
-        version_filename = Path(os.environ["IRONCLUST_PATH"]) / 'matlab' / 'version.txt'
+    @classmethod
+    def get_sorter_version(cls):
+        if cls.is_compiled:
+            version_filename = Path('/opt') / 'version.txt'
```

Comment:
I think this would be specific to running the compiled version in docker, but ideally one could also run the compiled version outside of a container. In case you have a compiled version, I'd return `"compiled"`

Reply:
Great! My first scratch was returning `matlab_compiled`, but I thought it'd be good to return the version that we have in the ironclust source file.
I'll change to `compiled`, sounds good

Thanks